openeo-geotrellis/src/test/scala/org/openeo/geotrellis/layers/FileLayerProviderTest.scala

```diff
@@ -40,7 +43,7 @@ object FileLayerProviderTest {
   // - BeforeClass
   // - AfterClass
   //
-  // This order feels arbitrary, so I made the code robust against order changes.
+  // This order is because of multiple Test framework versions, so I made the code robust against order changes.
```

Comment:
If the problem is that we're mixing JUnit 4 and 5 APIs and they don't play well together I would just get rid of the former and replace @BeforeClass with of @BeforeAll etc.